typescript/mcap/src/v0/ChunkBuilder.ts

```diff
@@ -69,6 +71,14 @@ class ChunkBuilder {
     this.recordWriter.writeMessage(message);
   }
 
+  finish(): void {
+    if (this.messageIndices) {
+      for (const index of this.messageIndices.values()) {
+        index.records.sort(([logTimeA], [logTimeB]) => Number(logTimeA - logTimeB));
```

Comment:
Do message index records need to be sorted? This seems like extra work that doesn't need to happen? The reader side should be doing the sorting.

Reply:
I thought this was part of the spec but you're right, it is not